components/icons/PadLock.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import Icon, { icon } from './Icon.jsx'
+
+const PadLock = (hexColor, props) => (
+  <Icon stroke={hexColor} strokeWidth='2' strokeLinecap='round' {...props}>
+    <g fill={hexColor} stroke='none' transform='translate(24, 18)'>
+      <path d='M49.5946415,45.2830189 C49.5946415,52.2656604 44.3395472,57.7358491 37.6331321,57.7358491 L16.1236981,57.7358491 C9.10709434,57.7358491 3.17954717,52.0335849 3.17954717,45.2830189 L3.17954717,28.3018868 L49.5946415,28.3018868 L49.5946415,45.2830189 Z M8.83992453,18.1132075 C8.83992453,9.22528302 16.5380377,2.26415094 26.3655849,2.26415094 L27.3912453,2.26415094 C36.4971907,2.26415094 43.231813,8.27981237 43.8826067,16.7548945 C43.9168645,17.2010234 43.9342642,17.6539674 43.9342642,18.1132075 C43.9342642,18.7392453 44.4403019,19.245283 45.0663396,19.245283 C45.6923774,19.245283 46.1984151,18.7392453 46.1984151,18.1132075 C46.1984151,7.61773585 38.2886038,0 27.3912453,0 L26.3655849,0 C15.2689811,0 6.57577358,7.95622642 6.57577358,18.1132075 L6.57577358,26.0377358 L0.915396226,26.0377358 L0.915396226,45.2830189 C0.915396226,53.2607547 7.87879245,60 16.1236981,60 L37.6331321,60 C45.6097358,60 51.8587925,53.5358491 51.8587925,45.2830189 L51.8587925,26.0377358 L8.83992453,26.0377358 L8.83992453,18.1132075 Z'></path>
+      <rect x='43.9358463' y='18.1132075' width='2.26415094' height='7.9245283'></rect>
+      <rect x='26.0377358' y='36.2264151' width='2.26415094' height='12.4528302'></rect>
+      <rect x='24.9056604' y='35.0943396' width='4.52830189' height='4.52830189' rx='2.26415094'></rect>
```

Comment:
I suppose after the 1st or 2nd number after . there will be no diff to the human eye, even in a x8 display. We have been asking our designer to pass SVGs over https://jakearchibald.github.io/svgomg/, just to make sure.